Flutter/pdf-viewer/form-filling.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: Form filling in Flutter PDF Viewer widget | Syncfusion
+description: Learn here all about form filling feature of Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: Flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Form filling in Flutter PDF Viewer (SfPdfViewer)
+
+The [SfPdfViewer](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer-class.html) allows you to fill, edit, save, export, and import the AcroForm fields in a PDF document.
+
+## Supported form fields
+
+You can load and fill in the following form fields in a PDF document using the Flutter PDF Viewer. 
+
+* Text box. 
+* Checkbox. 
+* Radio button. 
+* Combo box. 
+* Signature. 
+
+## Save form data 
+
+You can save the modified form fields data by calling the `saveDocument` method. Refer to the following code example. 
+
+{% tabs %}
+{% highlight dart hl_lines="15 16" %}
+
+final PdfViewerController _pdfViewerController = PdfViewerController(); 
+ 
+@override 
+Widget build(BuildContext context) { 
+  return Scaffold( 
+    appBar: AppBar( 
+      title: const Text('Syncfusion Flutter PDF Viewer'), 
+      actions: <Widget>[ 
+        IconButton( 
+          icon: const Icon( 
+            Icons.save, 
+            color: Colors.white, 
+          ), 
+          onPressed: () async { 
+            final List<int> savedBytes = 
+                await _pdfViewerController.saveDocument(); 
+          }, 
+        ), 
+      ], 
+    ), 
+    body: SfPdfViewer.asset( 
+      'assets/form_document.pdf', 
+      controller: _pdfViewerController, 
+    ), 
+  ); 
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N> When the `saveDocument` method is called, the document will be automatically reloaded after the save in the viewer. Also, the signature field will be flattened on save irrespective to the `PdfFlattenOption` provided.   
+
+### Flattening the form data on save 
+
+Flattening PDF form is a process of removing the form fields in the PDF document, thereby rendering the form fields appearance and its content in the page graphics. This will avoid the PDF form being edited in any device. Flutter PDF Viewer supports flattening the PDF form when saving. You can perform this action by setting the `PdfFlattenOption` to `PdfFlattenOption.formFields` in the `saveDocument` method.  
```

Comment:
Flattening PDF form is a process of removing the form fields in the PDF document, thereby rendering the form fields' appearance and content in the page graphics. This will avoid the PDF form from being edited on any device. Flutter PDF Viewer supports flattening the PDF form when saving. You can perform this action by setting the `PdfFlattenOption` to `PdfFlattenOption.formFields` in the `saveDocument` method.